qutebrowser/test/mainwindow/conftest.py

```diff
@@ -0,0 +1,34 @@
+"""
+pytest fixtures and utilities for testing.
+
+Fixtures defined here will be visible to all test files in this directory and
+below.
+"""
+
+import pytest
+
+from qutebrowser.config.config import ConfigManager
+from qutebrowser.utils import objreg
+
+
+@pytest.yield_fixture
+def default_config():
```

Comment:
Just as a note, this fixture currently is used in a single test (`test_progress.py`), so it could have been declared in that file instead just as easily, but I feel that this will be useful for more tests to come so decided already create the first `conftest.py`. :smile: 

This is another nice feature of fixtures, you can define them initially where they are needed, and move them to a `conftest` file if needed **without changing test code**. :wink: